src/test/resources/jcasc/elasticSearch.yaml

```diff
@@ -0,0 +1,26 @@
+jenkins:
```

Comment:
Only having everything under `unclassified` would be enough for testing, `jenkins` is not mandatory.